openapi.json

```diff
@@ -644,20 +644,24 @@
             "name": "clusterId",
             "in": "path",
             "required": true,
+            "description": "ID of the cluster which must conform to UUID format",
             "schema": {
               "type": "string",
               "minLength": 36,
               "maxLength": 36,
               "format": "uuid"
-            }
+            },
+            "example": "34c3ecc5-624a-49a5-bab8-4fdc5e51a266"
           },
           {
             "name": "ruleId",
             "in": "path",
             "required": true,
+            "description": "Enables a rule (ruleId) for cluster (clusterId) for current organization/user",
```

Comment:
this does not look like a proper description

Reply:
Updated. Copy&paste error